database/migrations/2022_07_27_005600_create_adoptions_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAdoptionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('adoptions', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('owner_id')->constrained();
+            $table->foreingId('pet_id')->constrained();
+            $table->foreingId('stackholder_id')->constrained();
+            $table->date('visit_date');
+            $table->date('adoption_date');
+            $table->longText('adoption_date');
```

Comment:
field `adoption_date` isnt `longText` type. Put `adoption_review` instead. But i was thinking both the owner and the stackholder should give a feedback on the adoption, so it can be like  `owner_review` and `stackholder_review`. What do you think?

Reply:
Yeah, both of them should be able to give some feedbacks. I will fix that and create a new pull request. 

Thanks!